### ISSUE TITLE
HTTPS to prevent the bookmarklet being blocked by browsers?

### ISSUE THREAD

**Issue by Flamekebab** [pinry/pinry#107]
This may already exist but need enabling somewhere - HTTPS support. The main reason I ask is because it seems that the bookmarklet works fine on sites that don't use HTTPS but as soon as one tries to use the script on a secure site browsers block the unsecured script. It's possible to override the blocking but it does render the bookmarklet nearly unusable.

**Comment by overshard** [comment]
This is up to you to implement and not Pinry. It could be something we are clearer about in the documentation though. There is a lot of documentation out there on how to implement HTTPS/SSL/TLS on a Django website with however you are deploying.

**Comment by Flamekebab** [comment]
Right-o, it's been a little while but I've tinkered with things and had some success.

I followed [this guide](http://www.marinamele.com/2014/09/security-on-django-app-https-everywhere.html) and added a few things to settings.py

I also made sure mod_wsgi was installed in Apache and setup a subdomain virtualhost thingy for Pinry with this in the file just after "<VirtualHost *:443>".

```
WSGIDaemonProcess pinry python-path=/var/opt/pinry:/var/opt/pinry/.venv/lib/python2.7/site-packages
WSGIProcessGroup pinry
WSGIScriptAlias / /var/opt/pinry/pinry/wsgi.py
```

I also used CertBot from [Let's Encrypt](https://letsencrypt.org/) to get a certificate for HTTPS and set that up.

Once that was done things seem to have worked. I packaged up the bookmarklet as a Chrome extension using [this site](https://sandbox.self.li/bookmarklet-to-extension/) and now all seems to be well.
